styles/main.css

```diff
@@ -0,0 +1,129 @@
+* {
+	box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+}
+
+html {
+	scroll-behavior: smooth;
+}
+
+.grid-container {
+	display: grid;
+	gap: 0;
+	grid-template-areas:
+		'h m m m'
+		'n m m m'
+		'n m m m'
+		'n f f f';
+}
+
+header {
+	grid-area: h;
+	height: 10vh;
+	width: 20vw;
+	padding-top: 4%;
+	display: flex;
+	justify-content: center;
+	position: sticky;
+	top: 0;
+}
+
+nav {
+	grid-area: n;
+	width: 20vw;
+	display: flex;
+	justify-content: center;
+	font-family: 'Roboto', sans-serif;
```

Comment:
You could pull any `font-family` properties out of individual element styles and unify it at the top with an `* { }`. That way, we can update the font in a single place easily.